src/Locale/en/Common.json

```diff
@@ -89,5 +89,19 @@
   "save": "Save",
   "select": "Select",
   "lsg": "Lsg",
-  "delete": "DELETE"
+  "delete": "DELETE",
+  "delete_norm":"Delete",
```

Comment:
let the uppercase delete be modified by adding `uppercase` class instead, to avoid confusion of usage of keys.

```suggestion
  "delete": "Delete",
```

Reply:
Hey i have done the changes